src/components/TldrawCanvas.tsx

```diff
@@ -219,17 +215,9 @@ const createDiscourseNode = async ({
   text: string;
   nodes?: DiscourseNode[];
 }) => {
-  const nodeTree = getFullTreeByParentUid(type).children;
-  const template = getSubTree({
-    tree: nodeTree,
-    key: "template",
-  }).children;
-  const stripUid = (n: RoamBasicNode[]): InputTextNode[] =>
-    n.map(({ uid, children, ...c }) => ({
-      ...c,
-      children: stripUid(children),
-    }));
-  const tree = template.length ? stripUid(template) : [{ text: "" }];
+  // This solves for blck-type and creates block in the DNP
+  // but could have uninteded consequenses for other defined discourse nodes
+  // also TODO: create in `Auto generated from ${title}`
   const specification = nodes.find((n) => n.type === type)?.specification;
   if (
```

Comment:
Is the use case for the solely for solving for handling `blck-type` discourse nodes?